src/locales/en/translation.json

```diff
@@ -9,8 +9,7 @@
     "evmTokens": "EVM tokens",
```

Comment:
I disagree with removing duplicates. We should do the opposite and split reused translation strings by semantic context.